packages/cf-component-radio/src/Radio.js

```diff
@@ -1,6 +1,86 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  cursor: theme.cursor,
+  display: theme.display,
+  minHeight: theme.minHeight,
+  paddingLeft: theme.paddingLeft,
+  marginTop: theme.marginTop,
+
+  '&:first-child': {
```

Comment:
Maybe, this is all copy-pasted and I'm not actually writing this code with any active thoughts other than trying to get the visuals to match. I can take a look at this but am unsure to what the problem is that I'm supposed to solve. Maybe explaining that would help me do the suggested change :)